aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -371,9 +394,36 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         }
     }
 
+    fun getHeadingMenu(): PopupMenu? {
+        return headingMenu
+    }
+
+    fun getListMenu(): PopupMenu? {
+        return listMenu
+    }
+
+    fun getSelectedHeadingMenuItem(): TextFormat? {
+        if (headingMenu?.menu?.getItem(0)?.isChecked!!) return TextFormat.FORMAT_PARAGRAPH
```

Comment:
That's a good idea.  I changed from integer literal references with `getItem()` to item identification references with `findItem()` in d5a4838bd92892492fece477cbd9fa5c4ed375f7.